Tests/MoyaSugarTests/Endpoint+Parameters.swift

```diff
@@ -0,0 +1,25 @@
+import Moya
+
+// Extension for Testing
+extension Endpoint {
```

Comment:
We can probably get rid of this.

Reply:
How do we find the way to test `parameters` and `parameterEncoding`?

Reply:
How about testing the case of `task`?